screens/Settings/DataBackupAndRestore.tsx

```diff
@@ -76,7 +76,7 @@ export const DataBackupAndRestore: React.FC = ({} = () => {
         </ListItem>
       </Pressable>
 
-      {controller.isSigningInFailed && (
```

Comment:
is the removed condition and new one same?

Reply:
No

Reply:
Could you please explain this up once @abhip2565 ?

Reply:
Connected with Abhishek on this